src/views/organisationType.njk

```diff
@@ -46,9 +46,33 @@
     {{ govukButton({
     text: "Continue",
     type: 'submit'
-   
   }) }}
 
   </form>
 
+  <script>
+    document.addEventListener("DOMContentLoaded", function () {
+      const storedOrganisationType = localStorage.getItem("organisationType");
+      const radioToSelect = document.querySelector('input[value=' + storedOrganisationType + ']');
```

Comment:
If this line is about selecting the field, it's preferable to use "document.getElementsByName" instead to query based on the input value.
Generally document.getElementsByName or document.getElementById are simpler selector for any element of the page.
doc: https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByName

Reply:
Interesting! Thanks Fabs, i didn't know this - we don't currently have names on the input fields, so rather than adding these in for just this purpose - i'm going to grab by the element ID instead